### PR TITLE
Add operf-macro package

### PR DESCRIPTION
**Pull request by Michaaell**

### CODE REVIEW

File: packages/operf-macro/operf-macro.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "operf-macro"
+maintainer: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+authors: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+homepage: "http://www.typerex.org/operf-macro.html"
+bug-reports: "http://github.com/OCamlPro/operf-macro/issues"
+license: "GPLv2"
```
Comment (yallop) [inline packages/operf-macro/operf-macro.0.2/opam]: GPLv2 ~> [MIT](https://github.com/OCamlPro/operf-micro/blob/master/LICENSE)

**Comment by chambart** [comment]
This seems to be working. The failing build is due to some connection problems.

**Comment by yallop** [comment]
I've restarted the failing build.

**Comment by chambart** [comment]
Thanks. By the way, is there a way for the sender of the pull request to restart a build ?

**Comment by nojb** [comment]
I believe you can do it by force-pushing again.

**Pull request merged**